charts/alfresco-transform-service/templates/_helpers-name.tpl

```diff
@@ -1,69 +1,44 @@
-{{- define "alfresco-transform-service.config-filestore.name" -}}
-{{- $scope := (dict "Values" (dict "nameOverride" "filestore-configmap" ) "Chart" .Chart "Release" .Release) }}
+{{- define "alfresco-transform-service.messagebroker.name" -}}
+{{- $scope := (dict "Values" (dict "nameOverride" "messagebroker" ) "Chart" .Chart "Release" .Release) }}
```

Comment:
did I risk a conflict when used under the umbrella chart?

Reply:
strictly speaking no as this name is not used elsewhere for now but there is no reason why such a cm would be used by this chart rather than another which also leverages the activemq component.